### ISSUE TITLE
Zu wenig Informationen

### ISSUE THREAD

**Issue by binfalse** [wahlbilanz/DeinWal.de#14]
Vielen Nutzern fehlt das noetige Vorwissen zu einigen Fragen, eg

* https://www.reddit.com/r/de/comments/6ug9ui/deinwalde_wahlomat_mit_bisherigen_abstimmungen/dlshyxl/
* https://twitter.com/asga404/status/898629809219158017
* https://twitter.com/Stoertebeker480/status/898822881261506560
* spon

Wir sollten mehr (optionale) Informationen mitliefern!

**Comment by tomthe** [comment]
Ja, sehe ich auch so. Gleichzeitig darf der normale Fragentext aber nicht weiter anschwellen. Wir sollten vielleicht ein optionales, aufklappbares informationsFeld einbauen.
Bloß wer schreibt dann die ganzen notwendigen Informationen da rein?

**Comment by binfalse** [comment]
Prinzipiell implementiert in  fc11584 7c186dd b9b501e ce8bca3 86add2c d692c24

Muessen vielleicht nochmal Informationen sammeln und entscheiden was wir rein nehmen?

**Comment by binfalse** [comment]
@tomthe guck mal https://deinwahl.de/quiz/1 -> klick auf "Mehr Informationen"

**Comment by tomthe** [comment]
Ja, genau so! 
Aber es sollte vielleicht visuell erkennbar sein, das man da drauf klicken kann. Also entweder dieser Pfeil/Dreieck oder unterstrichen wie ein ___Link___

**Comment by jzaefferer** [comment]
Die zusätzlichen Links sind nützlich, aber bei einzelnen Fragen ist es immer noch sehr schwierig. Zum Beispiel bei "Immer mehr neue Arbeitsverträge sind nur befristet. Sollten sachgrundlose Befristungen abgeschafft werden? " - auf der verlinkten Wikipedia Seite wird "sachgrundlos" zwar 33 mal erwähnt, aber ob das nun gut ist oder nicht ist mir immer noch nicht klar. Vielleicht wären Zeitungsberichte zu den einzelnen Fragen nützlich? Oder spricht da etwas gegen?

**Comment by binfalse** [comment]
Ne, super Idee.. Ich hatte bisher nur als Hotfix ein paar Links eingefuegt; mir war aber schon klar, dass das noch lang nicht ausreicht.

Bei Zeitungsartikeln ist halt wieder das Problem, dass die natuerlich moeglichst neutral Berichten sollten... Wenn du Ideen hast: raus damit! ;-)
